README.md

```diff
@@ -37,7 +37,10 @@ Have questions? Feel free to post them in [Forest Q&A](https://github.com/ChainS
 
 No need to install Rust toolchain or other dependencies, you will need only Docker.
 ```
-❯ docker run --init -it ghcr.io/chainsafe/forest:latest --help
+# daemon
+❯ docker run --init -it --rm ghcr.io/chainsafe/forest:latest --help
+# cli
+❯ docker run --init -it --rm --entrypoint forest-cli ghcr.io/chainsafe/forest:latest --help
```

Comment:
We need to use `--entrypoint forest-cli` to invoke cli from docker because we're using `ENTRYPOINT` instruction, should we switch to `CMD`? @lemmih @LesnyRumcajs

Reply:
Sounds good to me. We may need to potentially update many other things, including daily snapshots and sync checks. Let's do this in a separate issue.